NitroxModel/packages.config

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Autofac" version="2.6.3.862" targetFramework="net35" />
-  <package id="LiteNetLib" version="0.8.2-beta" targetFramework="net35" />
-  <package id="NLog" version="4.7.0" targetFramework="net40" />
+  <package id="NLog" version="4.7.0" targetFramework="net40" />  <package id="log4net" version="2.0.8" targetFramework="net35" />
```

Comment:
Is it right to add this log4net package?

Reply:
no, it is replaced with nlog. Its likely a merge mistake.